noggin/tests/unit/controller/test_user.py

```diff
@@ -81,6 +89,20 @@ def test_user_edit_post(client, logged_in_dummy_user):
     )
 
 
+@pytest.mark.vcr()
+def test_user_edit_post_minimal_values(client, logged_in_dummy_user):
+    """Test posting to the user edit page: /user/<username>/settings/profile/
+        with the bare minimum of values """
+    result = client.post('/user/dummy/settings/profile/', data=POST_CONTENTS_MIN)
+    print(result.data)
```

Comment:
Looks like a debug leftover... (I usually run a `git grep print` before pushing a PR to check for those, I always forget some)